modules/Installer/AWS.Tools.Installer.psm1

```diff
@@ -1,7 +1,34 @@
+Microsoft.PowerShell.Core\Set-StrictMode -Version 3
```

Comment:
I lifted the practice of prefixing `Set-StrictMode` from the `PowerShellGet` code. I will add prefixes to some of the other cmdlet calls as well (especially those from `PowerShellGet`). I don't think I want to add prefixes to `Select-Object`, `Where-Object`, `Write-Verbose`, etc. because it would affect readability of the code.